acceptance-tests/acceptance-tests-checkerframework/src/test/groovy/io/github/ascopes/jct/acceptancetests/checkerframework/CheckerNullTest.groovy

```diff
@@ -25,7 +25,7 @@ import org.junit.jupiter.api.parallel.Execution
 import org.junit.jupiter.api.parallel.ExecutionMode
 
 import static io.github.ascopes.jct.assertions.JctAssertions.assertThatCompilation
-import static io.github.ascopes.jct.pathwrappers.RamDirectory.newRamDirectory
+import static io.github.ascopes.jct.workspaces.impl.RamDirectory.newRamDirectory
```

Comment:
- [ ] Exclude impl types from JavaDocs.
- [ ] Update example snippet in `README.md` and `module-info.java`.